friends/src/App.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import FriendsList from './Components/FriendsList';
+import './App.css';
+import FriendForm from './Components/FriendForm';
+
+function App() {
+  return (
+    <div className="App">
+     <h2>Friends</h2>
```

Comment:
I would recommend using a class component here instead (with `state` and life cycle methods utilized) so that you can access inputs from `<FriendsForm />` (for adding new friend objects) and then add the information as a new entry in the `friends` object array to be used by `<FriendsList />`.